app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -350,18 +350,18 @@ class ThreadFragment : Fragment() {
         mainViewModel.toggleLightThemeForMessage.observe(viewLifecycleOwner, threadAdapter::toggleLightMode)
     }
 
-    private fun observeThreadLive() = with(binding) {
+    private fun observeThreadLive() = with(threadViewModel) {
 
-        threadViewModel.threadLive.observe(viewLifecycleOwner) { thread ->
+        threadLive.observe(viewLifecycleOwner) { thread ->
 
             if (thread == null) {
                 twoPaneViewModel.closeThread()
                 return@observe
             }
 
-            threadSubject.movementMethod = LinkMovementMethod.getInstance()
+            binding.threadSubject.movementMethod = LinkMovementMethod.getInstance()
```

Comment:
Shouldn't this line be called only once during the onViewCreated and not every time we observe the thread?

Reply:
Prbbly, yes.